### PR TITLE
feat: add authors details logic

### PR DESCRIPTION
**Pull request by smmd**
Add author detail page logic, I use my real information.

image

### CODE REVIEW

File: es/_posts/2019-01-25-ejemplo-blog-post.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: post
 title: "Ejemplo de blog post"
 date: 2019-01-25 16:42:00 -0500
 categories: programming
-author: "Jaimito El Cartero"
+author: jamie_the_mailman
```
Comment (klaussilveira) [inline es/_posts/2019-01-25-ejemplo-blog-post.md]: This needs to support an array for articles authored by multiple people.

Comment (smmd) [reply es/_posts/2019-01-25-ejemplo-blog-post.md]: @Briones help me to solve this here https://github.com/LinioIT/linioit.github.io/pull/11 please could you check his PR and then I could implement my current changes 😄

File: _data/authors.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Author details.
+jamie_the_mailman:
+  name: Jaimito El Cartero
+  email: jamie@linio.com
+  github: https://github.com/jamiemailman
+  twitter: http://twitter.com/jamiemailman
+  image: author.png
+jon_snow:
```
Comment (nesl247) [inline _data/authors.yml]: We should remove all the placeholder ones.

File: _posts/2019-01-25-example-blog-post.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: post
 title: "Example blog post"
```
Comment (nesl247) [inline _posts/2019-01-25-example-blog-post.md]: Do we need the example anymore?

Comment (smmd) [reply _posts/2019-01-25-example-blog-post.md]: I think we need because no post has been approved yet

Comment (smmd) [reply _posts/2019-01-25-example-blog-post.md]: I want to keep the example blog post until we merge a real publication. . . What do you think?

File: _data/authors.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Author details.
+sagrario_meneses:
+  name: Sagrario Meneses
+  nickname: smmd
+  email: sagrario.meneses@linio.com
+  github: https://github.com/smmd
+  twitter: http://twitter.com/sagmmd
+  image: smmd.png
```
Comment (klaussilveira) [inline _data/authors.yml]: No need to have assets for this. Just use gravatar.

File: _layouts/author.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: Author
+label: Author
+current: author
+---
+
+{% assign author = site.data.authors[page.author] %}
+<p class="blog-post-meta">
```
Comment (klaussilveira) [inline _layouts/author.html]: Can you include a screenshot in the PR so we can see how this looks?

Comment (smmd) [reply _layouts/author.html]: Done in the PR description 😄

**Pull request merged**